scipy/spatial/transform/tests/test_rotation.py

```diff
@@ -815,35 +783,30 @@ def test_as_euler_degenerate_compare_algorithms():
 
     idx = angles[:, 1] == 0  # find problematic angles indices
 
+    # Rotation of the form A/B/A are rotation around symmetric
+    # Axes
```

Comment:
This reads like an oddly placed linebreak in a sentence. If it's not, please reformulate or add a period/semicolon/whatever.
```suggestion
    # Rotation of the form A/B/A are rotation around symmetric;
    # Axes
```

Reply:
Should be fixed in the 3 occurences.